### ISSUE TITLE
GraphQL 1.9.9 breaks Client.load_schema

### ISSUE THREAD

**Issue by ericyd** [github/graphql-client#216]
GraphQL [release 1.9.9](https://github.com/rmosolgo/graphql-ruby/releases/tag/v1.9.9) broke the `load_schema` method in the client.

Examples to reproduce below

```
~ $ irb
2.4.2 :001 > require 'graphql'
 => true 
2.4.2 :002 > require 'graphql/client'
 => true 
2.4.2 :003 > require 'graphql/client/http'
 => true 
2.4.2 :004 > GraphQL::VERSION
 => "1.9.9" 
2.4.2 :005 > url = 'https://api.github.com/graphql'
 => "https://api.github.com/graphql" 
2.4.2 :006 > http = GraphQL::Client::HTTP.new(url)
 => #<GraphQL::Client::HTTP:0x00007fce605cd1a0 @uri=#<URI::HTTPS https://api.github.com/graphql>> 
2.4.2 :007 > schema = GraphQL::Client.load_schema(http)
RuntimeError: can't modify frozen GraphQL::Language::Nodes::Document
        from /Users/eric/.rvm/gems/ruby-2.4.2/gems/graphql-1.9.9/lib/graphql/language/nodes.rb:77:in `to_query_string'
        from /Users/eric/.rvm/gems/ruby-2.4.2/gems/graphql-client-0.14.0/lib/graphql/client/http.rb:68:in `execute'
        from /Users/eric/.rvm/gems/ruby-2.4.2/gems/graphql-client-0.14.0/lib/graphql/client.rb:77:in `dump_schema'
        from /Users/eric/.rvm/gems/ruby-2.4.2/gems/graphql-client-0.14.0/lib/graphql/client.rb:61:in `load_schema'
        from (irb):15
        from /Users/eric/.rvm/rubies/ruby-2.4.2/bin/irb:11:in `<main>'
```

**Comment by wowza-mccarthy** [comment]
Just to add to this issue since I am running into the same error:

```bundler: failed to load command: rspec (/Users/mac.mccarthy/.rbenv/versions/2.6.1/bin/rspec)
FrozenError: can't modify frozen GraphQL::Language::Nodes::Document
  /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-1.9.9/lib/graphql/language/nodes.rb:77:in `to_query_string'
  /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-client-0.14.0/lib/graphql/client/http.rb:68:in `execute'
  /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-client-0.14.0/lib/graphql/client.rb:77:in `dump_schema'
  /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-client-0.14.0/lib/graphql/client.rb:61:in `load_schema'```

What kind of chafed was that I couldn't point to a previous (working) version of graphql such as 1.8.11.

Forcing the gemfile to above version, so it looks like this:

```source "https://rubygems.org"

gem 'graphql-client'		'~>1.8.11'
gem 'rspec',                 '~>3.2.0'
gem 'nokogiri',              '~>1.8.3'
gem 'httparty',              '~>0.16.2'
gem "pry"
gem "dotenv"
gem "rspec_junit_formatter"```

Gets an error where it can't find the gem.

```Fetching gem metadata from https://rubygems.org/...........
Fetching gem metadata from https://rubygems.org/.
Could not find gem 'graphql-client~>1.8.11' in any of the gem sources listed in your Gemfile.```

Let me know if I'm barking up the wrong tree here, and should try to reach out to bundler. This dependency on version 1.9.9 breaks our circleCI processes during every build deployment. 

Thanks!

**Comment by epk** [comment]
> Just to add to this issue since I am running into the same error:
> 
> ```
> bundler: failed to load command: rspec (/Users/mac.mccarthy/.rbenv/versions/2.6.1/bin/rspec)
> FrozenError: can't modify frozen GraphQL::Language::Nodes::Document
>   /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-1.9.9/lib/graphql/language/nodes.rb:77:in `to_query_string'
>   /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-client-0.14.0/lib/graphql/client/http.rb:68:in `execute'
>   /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-client-0.14.0/lib/graphql/client.rb:77:in `dump_schema'
>   /Users/mac.mccarthy/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/graphql-client-0.14.0/lib/graphql/client.rb:61:in `load_schema'
> ```
> 
> What kind of chafed was that I couldn't point to a previous (working) version of graphql such as 1.8.11.
> 
> Forcing the gemfile to above version, so it looks like this:
> 
> ```
> source "https://rubygems.org"
> 
> gem 'graphql-client'		'~>1.8.11'
> gem 'rspec',                 '~>3.2.0'
> gem 'nokogiri',              '~>1.8.3'
> gem 'httparty',              '~>0.16.2'
> gem "pry"
> gem "dotenv"
> gem "rspec_junit_formatter"
> ```
> 
> Gets an error where it can't find the gem.
> 
> ```
> Fetching gem metadata from https://rubygems.org/...........
> Fetching gem metadata from https://rubygems.org/.
> Could not find gem 'graphql-client~>1.8.11' in any of the gem sources listed in your Gemfile.
> ```
> 
> Let me know if I'm barking up the wrong tree here, and should try to reach out to bundler. This dependency on version 1.9.9 breaks our circleCI processes during every build deployment.
> 
> Thanks!

You should be targeting `gem 'graphql', '~>1.8.11'` not 'graphql-client'

**Comment by wowza-mccarthy** [comment]
@epk You are correct! that was a typo on my behalf, i pasted the wrong info.

```
source "https://rubygems.org"

gem "graphql"				'~>1.8.11'	
gem 'graphql-client'	
gem 'rspec',                 '~>3.2.0'
gem 'nokogiri',              '~>1.8.3'
```
still gets me

Could not find gem 'graphql~>1.8.11' in any of the gem sources listed in your Gemfile.

**Comment by ericyd** [comment]
@wowza-mccarthy try adding a comma after `graphql` in your Gemfile, e.g.
```
...
gem "graphql",				'~>1.8.11'	
...
```

**Comment by wowza-mccarthy** [comment]
@ericyd That fixed the bundle issue for me, thanks so much! As an FYI, version 1.8.17 is the latest I can use without incurring the error mentioned at top post. Thanks everyone! my mistake

**Comment by ericyd** [comment]
`1.9.8` works for me too, though it looks like the issue will be resolved quickly enough

```
$ irb
2.4.2 :001 > require 'graphql'
 => true 
2.4.2 :002 > require 'graphql/client'
 => true 
2.4.2 :003 > require 'graphql/client/http'
 => true 
2.4.2 :004 > GraphQL::VERSION
 => "1.9.8" 
2.4.2 :005 > url = 'https://gitlab.com/api/graphql'
 => "https://gitlab.com/api/graphql" 
2.4.2 :006 > http = GraphQL::Client::HTTP.new(url)
 => #<GraphQL::Client::HTTP:0x00007fbcd2956ed8 @uri=#<URI::HTTPS https://gitlab.com/api/graphql>> 
2.4.2 :007 > schema = GraphQL::Client.load_schema(http)
 => #<GraphQL::Schema ...> 
```

**Comment by rmosolgo** [comment]
Should be fixed in graphql-client 0.15.0, please let me know if you have trouble with that version!

**Comment by ericyd** [comment]
LGTM - thank you for the quick response! 🙌
